.pre-commit-config.yaml

```diff
@@ -107,6 +103,13 @@ repos:
         args: [--line-length=120, --skip-errors]
         exclude: docs/source-app
 
+  - repo: https://github.com/codespell-project/codespell
```

Comment:
Seems this check cover all the files including readme, let remove the Azure spell check pipeline.   
And remove the invalid words in the inc_dict.txt.

Reply:
> Seems this check cover all the files including readme, let's remove the Azure spell check pipeline. And remove the invalid words in the inc_dict.txt.

The current detection directory does not contain the `./examples` folder

Reply:
Can we also add similar scope like spell check `${REPO_DIR}/examples/**/*.md|!${REPO_DIR}/examples/pytorch/**/huggingface_models/**/*.md|!${REPO_DIR}/examples/README.md`?     
Use another PR to do the change, and remove the Azure spell check pipeline.